src/util/tubemap.js

```diff
@@ -3919,6 +3932,73 @@ function compareReadsByLeftEnd2(a, b) {
   return 0;
 }
 
+// converts readPath to a CIGAR string
```

Comment:
What is `readPath` meant to be? I think it's a vg Path object expressed as a JS object, right?